### ISSUE TITLE
[Bug Report][3.4.9] v-textarea SSR hydration attribute mismatch

### ISSUE THREAD

**Issue by AlbertBrand** [vuetifyjs/vuetify#18959]
### Environment
**Vuetify Version:** 3.4.9
**Last working version:** 3.4.6
**Vue Version:** 3.4.3
**Browsers:** Chrome 120.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
- Add Vuetify to Nuxt 3 (default SSR mode)
- Add a `v-textarea` and bind a ref
- Let the page render and check the console:
```
[Vue warn]: Hydration attribute mismatch on <textarea class=​"v-field__input" rows=​"5" id=​"input-1" aria-describedby=​"input-1-messages">​Text in textarea​</textarea>​flex 
  - rendered on server: (not rendered)
  - expected on client: value="Text in textarea"
  Note: this mismatch is check-only. The DOM will not be rectified in production due to performance overhead.
  You should fix the source of the mismatch. 
  at <VField ref=Ref< undefined > style= {--v-textarea-control-height: ''} onClick=fn<onControlClick>  ... > 
  at <VInput ref=Ref< undefined > modelValue="Text in textarea" onUpdate:modelValue=fn<onUpdate:modelValue>  ... > 
  at <VTextarea modelValue="Text in textarea" onUpdate:modelValue=fn > 
  at <VApp> 
  at <App key=3 > 
  at <NuxtRoot>
```

### Expected Behavior
Correctly rendered/hydrated v-textarea.

### Actual Behavior
Missing hydrated attribute in v-textarea.

### Reproduction Link
[https://stackblitz.com/edit/nuxt-starter-2izca7?file=app.vue](https://stackblitz.com/edit/nuxt-starter-2izca7?file=app.vue)

**Comment by KaelWD** [comment]
Looks like a vue 3.4 bug, it doesn't happen with 3.3.13

**Comment by AlbertBrand** [comment]
Indeed, I upgraded Vue as well from 3.3.11 to latest.

**Comment by neutrino84** [comment]
I am seeing this with v-text-field as well

**Comment by MeleKun** [comment]
should I ignore the hydration mismatch coming from v-text-field or go back to older version of vue

**Comment by MeleKun** [comment]
> Should be fixed in vue 3.4.6

what do you mean o-0 am using vue 3.4.7 and it still has this issue

**Comment by KaelWD** [comment]
https://github.com/vuejs/core/pull/10011
